aerospike.template.conf

```diff
@@ -6,16 +6,17 @@ service {
 	group root
 	paxos-single-replica-limit 1 # Number of nodes where the replica count is automatically reduced to 1.
 	pidfile /var/run/aerospike/asd.pid
-	service-threads ${SERVICE_THREADS}
-	transaction-queues ${TRANSACTION_QUEUES}
-	transaction-threads-per-queue ${TRANSACTION_THREADS_PER_QUEUE}
+	# in order to use non-default options, replace ${SERVICE_THREADS}, etc. with a number
+	# service-threads ${SERVICE_THREADS}
+	# transaction-queues ${TRANSACTION_QUEUES}
+	# transaction-threads-per-queue ${TRANSACTION_THREADS_PER_QUEUE}
 	proto-fd-max 15000
 }
 
 logging {
 
 	# Log file must be an absolute path.
-	file ${LOGFILE} {
+	file /var/log/aerospike/aerospike.log {
```

Comment:
Why make this static?

Reply:
Annoyingly, the configuration file does not pick these up from the environment. There are 2 mutually exclusive ways to configure these parameters, i) use environment variables or ii) hardcode values in the conf file. The conf file is recommended but the default version of this file cannot be parsed as is, and thus I added some values for what was necessary and commented out the rest of the params (default values will be used for those)